server/src/resolvers/measurements.spec.js

```diff
@@ -0,0 +1,60 @@
+const { resolver } = require("graphql-sequelize");
+
+const measurements = require('./measurements');
+const data = require('../data');
+const models = require('../models');
+
+describe('measurements', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+    jest.restoreAllMocks();
+  });
+
+  describe('all', () => {
+    const root = {};
+    const args = {};
+    const context = {};
+    const info = {};
+    it('all', () => {
```

Comment:
These should have descriptions. Like,
``it('calls correct model and produces correct result`)``

Reply:
Fixed.